src/subr.c

```diff
@@ -299,7 +299,7 @@ void OP_subrcall(int subr_no, int argnum) {
       break;
     case sb_COLOR_BASE:
       POP_SUBR_ARGS;
-      /* retun DLword offsetbetween LISPBASE and Lisp_world */
+      /* return DLword offsetbetween LISPBASE and Lisp_world */
```

Comment:
Missing space in `offsetbetween`

Reply:
Thanks for catching that.  Must have scanned right over it as I went through the list.